core/trino-server-rpm/src/main/resources/dist/etc/init.d/trino

```diff
@@ -37,6 +37,15 @@ CONFIGURATION=(
     --server-log-file "${SERVER_LOG_FILE:-/var/log/trino/server.log}"
 )
 
+files_limit=131072
```

Comment:
Add a comment where this value comes from. Also add a comment in the docs where this is mentioned that the value needs to be updated here if changed there. (Or other way around - a comment here that value needs to be updated in docs if changed here).

Reply:
I did both. I built docs and checked that the comment in docs is _not_ rendered.